tests/k6/tests/performancetest_common/common.js

```
@@ -0,0 +1,31 @@
+    } catch (error) {
+        throw new Error(`Failed to load end users: ${error.message}`);
+    }
+    });
+    return endUsers;
+}
+
+export function getDefaultThresholds(counters, labels) {
+    let thresholds = {
+        http_req_failed: ['rate<0.01']
+    }
+    counters.flatMap(t => labels.map(l => thresholds[`${t}{name:${l}}`] = []));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)